it/v1/endpoints/RetrievePeriodSummaryControllerISpec.scala

```diff
@@ -194,37 +205,69 @@ class RetrievePeriodSummaryControllerISpec extends IntegrationBaseSpec {
           (AUTHORIZATION, "Bearer 123")
         )
     }
+  }
+  private trait TysTest extends Test {
+    override val periodId = "2023-04-01_2024-01-01"
+    val fromDate = "2023-04-01"
```

Comment:
Align the variables?

Reply:
Maybe run the whole thing through scalafmt: use "sbt scalafmtAll"